packages/resin-event-log/src/resin-event-log.js

```diff
@@ -61,14 +61,10 @@ module.exports = function(options) {
 		userId: null,
 		prefix: prefix,
 		start: function(user, callback) {
-			if (!user) {
-				return Promise.reject(new Error(
-					'user is required to start events interaction.'
-				)).asCallback(callback)
```

Comment:
@pimterry the idea was to add anon logins for all services with this PR, but I first want to do the one service and get feedback before tackling the others. If the approach is fine I'll do the remainder of the services (with tests).